wordpress-mobile/WordPress-iOS
# [Home Page Picker] Adds sample URL to the Choose a Domain Screen

**Fixes** https://github.com/wordpress-mobile/gutenberg-mobile/issues/3019

### To test:

1. Start the site creation flow (e.g. *Choose site > Add button*)
1. Select the *Create WordPress.com site* option
1. Press **Skip** or select a domain and press **Choose**
1. Verify that the sample URL view is shown
1. Search for something 
1. **Expect** the UI to disappear
1. Clear the results
1. **Expect** the UI to reappear

### Screenshots

Phone | Tablet
--- | ---
<a href="https://user-images.githubusercontent.com/3384451/108866978-b8d4ef80-75c2-11eb-9672-1952d76e7edd.png"><img width="300" src="https://user-images.githubusercontent.com/3384451/108866978-b8d4ef80-75c2-11eb-9672-1952d76e7edd.png"/></a> | <a href="https://user-images.githubusercontent.com/3384451/108867088-d2763700-75c2-11eb-9f64-06bfd8029d33.png"><img width="300" src="https://user-images.githubusercontent.com/3384451/108867088-d2763700-75c2-11eb-9f64-06bfd8029d33.png"/></a>
<a href="https://user-images.githubusercontent.com/3384451/108867150-e4f07080-75c2-11eb-9c04-74a0f6faaea6.png"><img width="300" src="https://user-images.githubusercontent.com/3384451/108867150-e4f07080-75c2-11eb-9c04-74a0f6faaea6.png"/></a> | <a href="https://user-images.githubusercontent.com/3384451/108867204-f33e8c80-75c2-11eb-8c3e-c866d9d6bf50.png"><img width="300" src="https://user-images.githubusercontent.com/3384451/108867204-f33e8c80-75c2-11eb-8c3e-c866d9d6bf50.png"/></a>

### PR submission checklist:
